Swift5_2.js

```diff
@@ -0,0 +1,111 @@
+class Swift5_2 extends PureComponent {
+
+  //
+  // Proptypes
+
+  static propTypes = {
+    regex: PropTypes.string.isRequired,
+    flags: PropTypes.string.isRequired,
+    delimiter: PropTypes.string.isRequired,
+    testString: PropTypes.string.isRequired,
+    isSubstituting: PropTypes.bool.isRequired,
+    isGlobal: PropTypes.bool.isRequired,
+    substString: PropTypes.string
+  };
+
+  //
+  // Control
+
+  _escapeSwiftString(input) {
+    // Simple search for a line break
+    let isMultiline = input.match(/\n/g)
+    // Wrap the string literal around # for each # found in the input string, plus 1 extra #
+    var hashes = "#"
+    while (input.search("#") != -1) {
```

Comment:
Why did you replace `input.search("hashes")` with `input.search("#")`?
Now if you type `##` and try to generate swift code, you will end up in an infinite loop.
This logic should add as many hashes as the maximum number of them in a row appears in a string.
For example `I'm using # and ## for this line` should be escaped as `###"I'm using # and ## for this line"###`

Reply:
Good catch! I initially thought the loop would iterate through the string character by character once, but I see it's not what it's doing now 😅 

Thanks for that! 🙌